cmake/wsrep.cmake

```diff
@@ -25,29 +25,10 @@ IF (NOT WITH_WSREP)
   RETURN()
 ENDIF()
 
-FILE(READ "${CMAKE_SOURCE_DIR}/WSREP_VERSION" wVER)
-STRING(REGEX MATCH "WSREP_VERSION_API=([0-9]*)" _ ${wVER})
```

Comment:
This code is originally from Codership. I think it is better to add a comment here that version check has been moved to `cmake/wsrep_version.cmake` so that we know where to look into in case of merge conflicts in future.

Reply:
Done.